Bio/AlignIO/NexusIO.py

```diff
@@ -140,75 +140,4 @@ def _classify_alphabet_for_nexus(self, alphabet):
             raise ValueError("Need a DNA, RNA or Protein alphabet")
 
 if __name__ == "__main__":
-    from Bio._py3k import StringIO
-    print("Quick self test")
-    print("")
-    print("Repeated names without a TAXA block")
-    handle = StringIO("""#NEXUS
-    [TITLE: NoName]
-
-    begin data;
-    dimensions ntax=4 nchar=50;
-    format interleave datatype=protein   gap=- symbols="FSTNKEYVQMCLAWPHDRIG";
-
-    matrix
-    CYS1_DICDI          -----MKVIL LFVLAVFTVF VSS------- --------RG IPPEEQ---- 
-    ALEU_HORVU          MAHARVLLLA LAVLATAAVA VASSSSFADS NPIRPVTDRA ASTLESAVLG 
-    CATH_HUMAN          ------MWAT LPLLCAGAWL LGV------- -PVCGAAELS VNSLEK----
-    CYS1_DICDI          -----MKVIL LFVLAVFTVF VSS------- --------RG IPPEEQ---X
-    ;
-    end; 
-    """)  # noqa for pep8 W291 trailing whitespace
-    for a in NexusIterator(handle):
-        print(a)
-        for r in a:
-            print("%r %s %s" % (r.seq, r.name, r.id))
-    print("Done")
-
-    print("")
-    print("Repeated names with a TAXA block")
-    handle = StringIO("""#NEXUS
-    [TITLE: NoName]
-
-    begin taxa
-    CYS1_DICDI
-    ALEU_HORVU
-    CATH_HUMAN
-    CYS1_DICDI;
-    end;
-
-    begin data;
-    dimensions ntax=4 nchar=50;
-    format interleave datatype=protein   gap=- symbols="FSTNKEYVQMCLAWPHDRIG";
-
-    matrix
-    CYS1_DICDI          -----MKVIL LFVLAVFTVF VSS------- --------RG IPPEEQ---- 
-    ALEU_HORVU          MAHARVLLLA LAVLATAAVA VASSSSFADS NPIRPVTDRA ASTLESAVLG 
-    CATH_HUMAN          ------MWAT LPLLCAGAWL LGV------- -PVCGAAELS VNSLEK----
-    CYS1_DICDI          -----MKVIL LFVLAVFTVF VSS------- --------RG IPPEEQ---X
-    ;
-    end; 
-    """)  # noqa for pep8 W291 trailing whitespace
-    for a in NexusIterator(handle):
-        print(a)
-        for r in a:
-            print("%r %s %s" % (r.seq, r.name, r.id))
-    print("Done")
-    print("")
-    print("Reading an empty file")
-    assert 0 == len(list(NexusIterator(StringIO())))
-    print("Done")
-    print("")
-    print("Writing...")
-
-    handle = StringIO()
-    NexusWriter(handle).write_file([a])
-    handle.seek(0)
-    print(handle.read())
-
-    handle = StringIO()
-    try:
-        NexusWriter(handle).write_file([a, a])
-        assert False, "Should have rejected more than one alignment!"
-    except ValueError:
-        pass
+    from Bio._utils import run_doctest
```

Comment:
This line doesn't do anything - you import the function ``run_doctest`` but don't use it:

``` python
from Bio._utils import run_doctest
```